ha/ha.go

```diff
@@ -120,17 +125,28 @@ func (h *HA) StartHA(chEnv chan *Environment) {
 	}
 }
 
-func (h *HA) waitForEnvironment(chEnv chan *Environment) {
+func (h *HA) waitForEnvironment(chEnv chan *Environment) *Environment {
 	// Wait for first heartbeat
 	env := <-chEnv
 	if env == nil {
 		log.WithFields(log.Fields{
 			"context": "HA",
 		}).Error("Received empty environment.")
 		h.super.ChErr <- errors.New("received empty environment")
-		return
+		return &Environment{}
 	}
+
+	return env
+}
+
+func (h *HA) setAndInsertEnvironment(env *Environment) error {
 	h.super.EnvId = env.ID
+
+	_, err := h.super.Dbw.SqlExec(mysqlObservers.insertIntoIcingadbInstance,
+		"REPLACE INTO environment(id, name) VALUES (?, ?)",
+		env.ID, env.Name)
```

Comment:
IMAO a such function call's `(` `)` and the params have to be on separate lines.